### ISSUE TITLE
ds.dataFrameSubset keep.cols / rm.cols parameter should accept character vectors?

### ISSUE THREAD

**Issue by tombisho** [datashield/dsBaseClient#297]
Reviewing the ds-helper package, a function is provided to return the index of a column in a dataframe, because ds.dataFrameSubset requires this for the keep.cols/rm.cols parameter.

It would be easier if ds.dataFrameSubset would accept the column name, because then this helper function would not be needed?

**Comment by davraam** [comment]
Yes we should modify the ds.dataFrameSubset to accept column names. This is important when the order of the columns is not the same across the different studies.

**Comment by tombisho** [comment]
That's a good point, I hadn't thought of that!

**Comment by tombisho** [comment]
Just to revisit this issue, one of our analysts is saying that the removal of ds.subset (where you can use column names) will cause a major issue because the columns are in different orders in each study. So I think ds.subset cannot be removed until this feature is implemented.
